File: Python/1026.py
def xor_bit_a_bit(n1, n2):
    print(n1 ^ n2)

def executar(i):
    n1, n2 = i.split(" ")
    xor_bit_a_bit(int(n1), int(n2))

while True:
    try:
        i = input()
        executar(i)
    except (EOFError):
        break

File: JavaScript/1026.js
// const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const input = require('fs').readFileSync('./stdin', 'utf8');
const lines = input.split('\n')

// XOR - bit
const obter_bit_a_bit = (n) => console.log(n[0] ^ n[1])

lines.forEach((e) => obter_bit_a_bit(e.split(' ').map((n) => Number(n))))

// Teste

// 4 6
// 6 9